src/katsdpcontroller/fake_servers.py

```diff
@@ -269,8 +269,58 @@ def __init__(self, *args, **kwargs) -> None:
         super().__init__(*args, **kwargs)
         channel_offset = self.get_command_argument(int, "--channel-offset-value")
         channels_per_substream = self.get_command_argument(int, "--channels-per-substream")
+        beam_outpus = self.get_command_arguments(_parse_key_val, "--beam")
+        beam_names = [beam["name"] for beam in beam_outpus]
         corrprod_outputs = self.get_command_arguments(_parse_key_val, "--corrprod")
         corrprod_names = [corrprod["name"] for corrprod in corrprod_outputs]
+
+        for beam_name in beam_names:
+            self.sensors.add(
+                Sensor(
+                    str,
+                    f"{beam_name}.chan-range",
+                    "The range of channels processed by this B-engine, inclusive",
+                    default=f"({channel_offset},{channel_offset + channels_per_substream - 1})",
+                    initial_status=Sensor.Status.NOMINAL,
+                )
+            )
+            self.sensors.add(
+                Sensor(
+                    str,
+                    f"{beam_name}.delay",
+                    "The delay settings of the inputs for this beam: (loadmcnt "
+                    "<ADC sample count when model was loaded>, delay <in seconds>,"
```

Comment:
```suggestion
                    ", delay [s],"
```

Again, I don't think this first part is necessary.
You can abbreviate the units for the delay, as well as phase as `[rad]` without a loss of information.